frontend/index.html

```diff
@@ -9,7 +9,7 @@
     <title>DedupliFHIR</title>
   </head>
   <body>
-    <div class="content">
+    <div class="content" style="width: 100%">
```

Comment:
Any reason for the inline style here as opposed to a one-off class?

Reply:
This styling is just specific to this content class div and won't be re-used anywhere else which is why I did not go with a one-off class